scripts/start-db-in-docker.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e -u -x
+
+cd /cf-networking-release
+export GOPATH=$PWD
+
+function loadIFB {
+  set +e
+    depmod $(uname -r)
+    modprobe ifb
+  set -e
+}
+
+function bootDB {
+  db=$1
+
+  if [ "$db" = "postgres" ]; then
+    launchDB="(docker-entrypoint.sh postgres &> /var/log/postgres-boot.log) &"
+    testConnection="psql -h localhost -U postgres -c '\conninfo' &>/dev/null"
+  elif [ "$db" = "mysql" ]  || [ "$db" = "mysql-5.6" ]; then
+    launchDB="(MYSQL_ROOT_PASSWORD=password /entrypoint.sh mysqld &> /var/log/mysql-boot.log) &"
+    testConnection="echo '\s;' | mysql -h 127.0.0.1 -u root --password='password' &>/dev/null"
+  else
+    echo "skipping database"
+    return 0
+  fi
+
+  echo -n "booting $db"
+  eval "$launchDB"
+  for _ in $(seq 1 60); do
+    set +e
+    eval "${testConnection}"
+    exitcode=$?
+    set -e
+    if [ ${exitcode} -eq 0 ]; then
```

Comment:
Can't this check be squashed only within the condition of the if statement?

In short can't we have it like `if "${testConnection}"; then echo "connection established to $db"; fi`?

Reply:
Did some testing, and we need to `if eval "${testConnection}"` to get all the parameters/quoting correct when executed, but it's still a little cleaner than it was.

Reply:
Yep, definitely! 👍